### PR TITLE
Changing notation for terminal ingredients

### PR DESCRIPTION
**Pull request by zanellia**
This PR changes the notation for terminal ingredients in the template-based interface from 'N' to '_e'. It is a first step towards merging of the "prototyping" and "embedded" interfaces.

### CODE REVIEW

**Comment by zanellia** [comment]
@giaf please have a look to see how close we are to having a unified description of the OCP for the two interfaces :)

File: interfaces/acados_template/acados_template/c_templates/acados_solver_sfun.in.c [hunk 1]
```diff
@@ -105,7 +105,7 @@ static void mdlInitializeSizes (SimStruct *S)
     // specify the direct feedthrough status 
     ssSetInputPortDirectFeedThrough(S, 0, 1); // current state x0
     ssSetInputPortDirectFeedThrough(S, 1, 1); // y_ref
-    ssSetInputPortDirectFeedThrough(S, 2, 1); // y_ref_N
+    ssSetInputPortDirectFeedThrough(S, 2, 1); // y_ref__e
```
Comment (giaf) [inline interfaces/acados_template/acados_template/c_templates/acados_solver_sfun.in.c]: here the is a double underscore

Comment (zanellia) [reply interfaces/acados_template/acados_template/c_templates/acados_solver_sfun.in.c]: wow, good catch! It's a comment, but still.

Comment (giaf) [reply interfaces/acados_template/acados_template/c_templates/acados_solver_sfun.in.c]: it just jumped into my eye

**Comment by giaf** [comment]
@zanellia the easiest is to have a kind of live presentation of our respective interfaces. 

Can we organize something? (Possibly also official and open to other group members)

**Pull request merged**